lib/drivers/insuletDriver.js

```diff
@@ -1512,7 +1520,15 @@ module.exports = function (config) {
       postrecords = buildBGRecords(data, postrecords);
       // first sort by log index
       postrecords = _.sortBy(postrecords, function(d) {
-        return d.index;
+        if (d.index) {
+          return d.index;
+        }
+        else if (d.payload && d.payload.logIndices) {
+          return d.payload.logIndices[0];
+        }
+        else {
```

Comment:
OK, because I shifted the deletion of `index` later, I actually didn't need to change the behavior of this sort. Going back to just sorting by `d.index` (about to push the reversion).